### ISSUE TITLE
Django utils not found in auto import

### ISSUE THREAD

**Issue by rchiodo** [microsoft/pylance-release#3815]
This should have found 'django.utils'

![image](https://user-images.githubusercontent.com/19672699/211420577-bad8ff9a-3581-4858-9b9b-7319a387252f.png)

Settings are here:

```json
  "python.analysis.packageIndexDepths": [
    {
      "name": "",
      "depth": 2,
      "includeAllSymbols": true
    },
    {
      "name": "pandas",
      "depth": 2
    },
    {
      "name": "django",
      "depth": 4,
      "includeAllSymbols": true
    }
  ],
  "python.analysis.autoImportCompletions": true,

```

**Comment by heejaechang** [comment]
looks like 
```
 {
      "name": "",
      "depth": 2,
      "includeAllSymbols": true
    },
```

this is messing things up. need to investigate.

**Comment by heejaechang** [comment]
This issue has been fixed in prerelease version 2023.1.31, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023131-18-January-2023-PreRelease)
